inc/class-ac-yoast-acf-content-analysis.php

```diff
@@ -212,6 +212,11 @@ protected function get_blacklist_type() {
 			$blacklist->remove( 'gallery' );
 			$blacklist->remove( 'repeater' );
 			$blacklist->remove( 'flexible_content' );
+		} elseif ( ! defined( 'ACF_PRO' ) || ! ACF_PRO ) {
```

Comment:
Can't this be simplified to just check on the `ACF_PRO` define?
The version does not seem to be relevant.

Reply:
:man_facepalming: